libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -810,7 +810,6 @@ - (void)dismissAuthViewControllerIfPresentForScene:(UIScene *)scene completion:(
 
 #pragma mark - SFOAuthCoordinatorDelegate
 - (void)oauthCoordinatorWillBeginAuthentication:(SFOAuthCoordinator *)coordinator authInfo:(SFOAuthInfo *)info {
-    coordinator.authSession.authInfo  = info;
```

Comment:
So this is to just remove unneeded reference?  Basically looks like we just call the SFOAuthCoordinator directly to the auth info? Seems like a good opportunity to convert to Swift 👍.

Reply:
yup! convert what part?